src/functionalTest/groovy/org/cadixdev/gradle/licenser/LicenserPluginFunctionalTest.groovy

```diff
@@ -29,14 +29,40 @@ import org.gradle.testkit.runner.TaskOutcome
 import org.junit.Rule
 import org.junit.rules.TemporaryFolder
 import spock.lang.Specification
+import spock.lang.Unroll
 
 import java.nio.file.Paths
 
 class LicenserPluginFunctionalTest extends Specification {
```

Comment:
Should we split up the tests in this file into multiple tests?
It's been growing quite substantially.

Reply:
Possibly -- I think I thought about this a bit I think, but there wasn't a super obvious division of tests. Perhaps check/format/other, or one file per test?

There's also discussion about moving this plugin away from Groovy -- such a change may make more sense in that batch of refactoring.